### PR TITLE
Check for quota

### LINKED ISSUE #1273: [QA] User with Quota 0 can create files and empty folders
Outside of any sharing portal, in the filesystem root folder, users can create folders, and create empty files, even after the owncloud administrator reduced user's quota to 0B.

This should 
a) trigger an error in owncloud core
b) should be suppressed in the frontend code, e.g. by removing the '+NEW' button.

### PR DESCRIPTION
**Pull request by LukasHirt**
## Description
Added check if user has not reached maximum of his quota to prevent from uploading or creating.

## Related Issue

- Fixes #1273

## How Has This Been Tested?

- test environment: Manually
- test case 1: Set quota "1 MB" and refresh Phoenix

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Add check in case user is uploading file which is bigger then space he has left

### CODE REVIEW

File: apps/files/src/store/actions.js [hunk 1]
```diff
@@ -145,6 +145,15 @@ export default {
       }, { root: true })
     }).finally(() => {
       context.commit('UPDATE_FOLDER_LOADING', false)
+      client.users.getUser(user.id).then(res => {
```
Comment (DeepDiver1975) [inline apps/files/src/store/actions.js]: do we need this extra call? afaik login shall return the same data ....

look here https://github.com/owncloud/phoenix/blob/b3ae5b66374ba40c6bf09767088cde9e703fd2e6/src/store/user.js#L40

Comment (DeepDiver1975) [reply apps/files/src/store/actions.js]: damn - the information is not in there ....

**Comment by DeepDiver1975** [comment]
should we display the quota somewhere?

**Comment by DeepDiver1975** [comment]
damn - wrong button ...

File: apps/files/src/store/actions.js (hunk 1 above)
Comment (LukasHirt) [reply apps/files/src/store/actions.js]: Yes, I looked there also and had the same revelation unfortunately

**Comment by LukasHirt** [comment]
> should we display the quota somewhere?

It would be nice to have it somewhere in FileList but it's already getting a little bit crowded there. Other option could be to display it in personal page but then it would be a little bit hidden.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIUpload</strong> failed. Please find the screenshots inside ...</summary><p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIUpload</strong> failed. Please find the screenshots inside ...</summary><p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>




</p></details>

**Pull request merged**